### ISSUE TITLE
'parentNode' of undefined Error

### ISSUE THREAD

**Issue by internette** [akfish/node-vibrant#76]
I'm attempting to use this module alongside React. I'm getting the current error:
```
Cannot read property 'parentNode' of undefined
    at BroswerImage../node_modules/node-vibrant/lib/image/browser.js.BroswerImage.remove
```

**Comment by Andsbf** [comment]
same here, interesting that if chrome devtools is open everything works nicely 😕

**Comment by crutchcorn** [comment]
I'm having the same problem. Any known fix for this?

**Comment by jan-scorl** [comment]
Same problem here with VueJS

**Comment by crutchcorn** [comment]
Can anyone test this out:
https://github.com/ceoss/node-vibrant/

Tell me if this is resolved for them? It is for my use-case in VueJS and Chrome, but I want to confirm that more widely before investigating much much further

I'll be working more on this and will be making PRs upstream (tho `master` on the fork will be a continued HEAD of all my changes to fix stuff until this repo gets a little more active so we can just `npm i` that git repo [remember to `npm i` a commit!])

**Comment by Andsbf** [comment]
Far from ideal, but the work around I have found is to reload the image instead let it read from cache, something in the following lines:

```
const getColorPalleteFromSrc = () => {
  const uniqueSrc = `${src}?q=${Date.now()}`;

  const palette = await  Vibrant.(uniqueSrc).getPalette();
}

**Comment by abarrafo** [comment]
I was having this issue. But, I also was receiving a CORS error on the image being loaded by node vibrant. This seems to have some to do with Chrome not allowing images it be loaded into canvas without an `access-control-allow-origin` header. In my case, I have been using react-palette and I simply had to update my image response to include this header and the parent node error is gone and everything works. 

I am hosting my images in S3 with a default CORS configuration. I then front it with CloudFront, lastly, I set up my CloudFront origins configuration to always include an origins header, thus forcing S3 to respond with the `access-control-allow-origin` header.

**Comment by crutchcorn** [comment]
I'm fairly certain that this is resolved by:
#61 to some degree

**Comment by crutchcorn** [comment]
Can someone who had this problem test against `3.1.1` and let us know if this is fixed? In my experience it had been but I'd like to confirm

**Comment by crutchcorn** [comment]
Should be fixed for sure in 3.1.2. In the extremely ~likely~ unlikely possibility this is not the case, please bump thread, trying to clear out issues :)
